SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -1224,11 +1224,20 @@ void USpatialSender::UpdateInterestComponent(AActor* Actor)
 	Connection->SendComponentUpdate(EntityId, &Update);
 }
 
+#if WITH_EDITOR
```

Comment:
"delete instead of tombstone" should only be required from the editor, but if you'd prefer I remove the #if WITH_EDITOR checks just let me know.